tests/a_scripts/inference/test_convert_composer_to_hf.py

```diff
@@ -152,6 +152,15 @@ def check_hf_tokenizer_equivalence(
     tokenizer1.__dict__['init_kwargs'].pop('vocab_file', None)
     tokenizer2.__dict__['init_kwargs'].pop('vocab_file', None)
 
+    # tokenizer.init_kwargs['merges_file'] is set when loading with AutoTokenizer.from_pretrained, but is set to
+    # None when you save and reload, so here we just check that its the same if it is present in both tokenizers.
```

Comment:
@dakinggg Are you familiar with the merges_file? Is it ok that it's None when you save and reload? I guess technically it had been missing completely until this transformers bump, so it's probably fine.

Reply:
yeah this should be ok. i think they changed in the patch release yesterday